src/frontend/src/common/styles/appStyle.ts

```diff
@@ -4,8 +4,4 @@ export const StyledApp = styled.div`
   height: 100vh;
   flex-flow: column wrap;
   justify-content: center;
-  :not(i) {
```

Comment:
[not(i) was added for semantic-ui's icon font](https://github.com/chanzuckerberg/czgenepi/pull/344).  We don't need this anymore, so removing.